tests/dummy/app/pods/details/controller.js

```diff
@@ -0,0 +1,47 @@
+import Ember from 'ember'
+
+export default Ember.Controller.extend({
+
+  headerComponent: 'fre-details',
+  baseRoute: 'details.maps',
+
+  subRoutes: [
```

Comment:
Once we have `ember-block-slots` available, I'd like to revisit this design, but let's move forward with this for now